src/components/Elements/Input.jsx

```diff
@@ -0,0 +1,10 @@
+import "../Expense/Expense.css";
```

Comment:
CSS files should be in the different folder. like `css`

Reply:
Yes we can move css file in different folder. 
There are may ways to use css. Every method have its own pros and cons.
1) Inline styling:
2) style.css
3) sass
4) styles.module.css
5) CSS-in-JS (Best way)